docs/examples/metallb-loadbalancer.md

```diff
@@ -0,0 +1,120 @@
+
+Port 7946 (TCP & UDP) must be allowed between the nodes.
+
+### Install MetalLB
+
+1. Install MetalLB using the official manifests by MetalLB
+
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.10.2/manifests/namespace.yaml
+    kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.10.2/manifests/metallb.yaml
+    ```
+
+2. Create ConfigMap for MetalLB
+
+    Next you need to create ConfigMap, which includes an IP address range for the load balancer. The pool of IPs must be dedicated to MetalLB's use. You can't reuse for example the Kubernetes node IPs or IPs controlled by other services. You can, however, use private IP addresses, for example 192.168.1.180-192.168.1.199, but then you need to take care of the routing from the external network if you need external access. In this example, we don't need it.
+
+    Create a YAML file accordingly, and deploy it: ```kubectl apply -f metallb-configmap.yaml```
+
+    ```YAML
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      namespace: metallb-system
+      name: config
+    data:
+      config: |
+        address-pools:
+        - name: default
+          protocol: layer2
+          addresses:
+          - <ip-address-range-start>-<ip-address-range-stop>
+    ```
+
+3. Deploy an example application (web server) with a load balancer
+
+    ```YAML
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: web
+    ---
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: web-server
+      namespace: web
+    spec:
+      selector:
+        matchLabels:
+          app: web
+      template:
+        metadata:
+          labels:
+            app: web
+        spec:
+          containers:
+          - name: httpd
+            image: httpd:2.4.48-alpine3.14
+            ports:
+            - containerPort: 80
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: web-server-service
+      namespace: web
+    spec:
+      selector:
+        app: web
+      ports:
+        - protocol: TCP
+          port: 80
+          targetPort: 80
+      type: LoadBalancer
+    ```
+
+4. Check your LoadBalancer
+
+    Run the following command to see your LoadBalancer with the external-ip and port.
+
+    ```shell
+    kubectl get service -n web
+    ```
+
+5. Access your example application
+
+    If you used private IP addresses for MetalLB in the ConfigMap (in step 2), you should run the following command from the local network. Use the IP address from the previous step.
+
+    ```shell
+    curl <EXTERNAL-IP>
+    ```
+
+    If you are successful, you should see ```<html><body><h1>It works!</h1></body></html>```.
+
+## Additional information
+
+For more information about MetalLB installation, take a look at [the official MetalLB documenation](https://metallb.universe.tf/installation/).
```

Comment:
`documenation` -> `documentation`

Reply:
I'll include this correction in another ongoing change.